### DISCUSSION TITLE
How to show remote repository?

### DISCUSSION THREAD

**Discussion by denisgolius** [starship/starship#2866] [Ideas]
I made some customization with `git-branch` but I need to show remote repository cuz I worked with many different own forks and I will be glad to see remote repository with bash. I don't want to use zsh and maybe there is something that I missed in the docs?

![image](https://user-images.githubusercontent.com/5650611/124614126-9ad6a000-de7c-11eb-9fde-da1a9032454c.png)

**Comment by zanna-37** [comment]
Maybe this custom module can be helpful.
https://github.com/starship/starship/discussions/1252#discussioncomment-194627
